src/sentry/api/analytics.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import, print_function
+
+from sentry import analytics
+
+
+class OrganizationSearchCreatedEvent(analytics.Event):
+    # HELP is organization-search a reasonable name here or should it just be saved searches
+    type = 'organization-search.created'
+
+    attributes = (
+        analytics.Attribute('organization_id'),
+        analytics.Attribute('type'),
+        analytics.Attribute('id'),
+        analytics.Attribute('user_id'),
+    )
+
+
+class OrganizationSearchDeletedEvent(analytics.Event):
+    # HELP is organization-search a reasonable name here or should it just be saved searches
+    type = 'organization-search.deleted'
```

Comment:
Have these events been added to reload?

Reply:
For my reference this is what Mark is referring to: https://github.com/getsentry/reload/blob/master/reload_app/events.py and no. But now that I know I should I will. :)